web/src/components/storage/VolumeForm.jsx

```diff
@@ -240,7 +240,7 @@ const SizeOptions = ({ errors, formData, volume, onChange }) => {
   const { sizeMethod } = formData;
   const sizeWidgetProps = { errors, formData, volume, onChange };
 
-  const sizeOptions = [SIZE_METHODS.MANUAL, SIZE_METHODS.RANGE];
+  const sizeOptions = [SIZE_METHODS.RANGE, SIZE_METHODS.MANUAL];
 
   if (volume.adaptiveSizes) sizeOptions.unshift(SIZE_METHODS.AUTO);
```

Comment:
Another option could be going from "Fixed" to "Auto". I.e., 


```suggestion
  const sizeOptions = [SIZE_METHODS.MANUAL, SIZE_METHODS.RANGE];

  if (volume.adaptiveSizes) sizeOptions.push(SIZE_METHODS.AUTO);
```

| Proposed now | Another proposal |
|-|-|
| ![Screen Shot 2023-06-13 at 07 47 30](https://github.com/openSUSE/agama/assets/1691872/cc56cadc-5e6a-47d3-98a0-80b9272a1e75) | ![Screen Shot 2023-06-13 at 07 56 52](https://github.com/openSUSE/agama/assets/1691872/7508bda7-3080-4750-9edb-2160f66248b4) |

Reply:
Now that you mention it, I would vote for "Fixed", "Range", "Auto". Note that "Auto" is only shown for some volumes (typically root) so the other two options will always be at the same position independently on the presence of "Auto".